### ISSUE TITLE
Snake can't claim square

### ISSUE THREAD

**Issue by cdiener1** [IamVaibhavsar/Snake_Game#1]
After claiming about 5, the snake refused to catch this one. Here I have the screenshot of it driving right over it.

https://user-images.githubusercontent.com/48371916/56462234-ad448c00-638d-11e9-9f1c-b9c160060c27.png

**Comment by IamVaibhavsar** [comment]
Hii Cdiener1,

First of all thank you for playing this game and giving the feedback.

I am aware of this bug, may it causes because the size of the square
doesn't matches with the resolution. Thanks why sometimes snake overlaps on
it.

I tried to figure it out, but I couldn't!

You can also check out my TO DO APP on my github profile and let me know
how do you like it!

Thank you once again!

**Comment by cdiener1** [comment]
Is both the width and the height of the screen divided by the with of the snake's head an integer. If not, that would explain the porblem.

**Comment by cdiener1** [comment]
Sorry! My spelling just died. Let me try again. If both the width and the height of the screen, divided by the with of the snake's head, is not an integer, that would explain the problem. Just multiply the width of the snake's head by whatever integer you want and use that to reset the width of the playing field.
